repos/fdbt-site/src/pages/api/managePurchaseMethod.ts

```diff
@@ -86,12 +93,59 @@ export default async (req: NextApiRequestWithSession, res: NextApiResponse): Pro
                     id: 'purchase-method-name',
                 });
             }
+
+            if (id) {
+                const purchaseDetails = await getSalesOfferPackageByIdAndNoc(id, noc);
+                if (purchaseDetails.isCapped) {
+                    const dbIsCapped = purchaseDetails.isCapped;
```

Comment:
We're saying here, if (purchaseDetails.isCapped) and if that is true, we enter the if block.

Now we know its true, we dont need to assign 'true' to a variable called dbIsCapped.